app/services/validation_beta_service.rb

```diff
@@ -25,13 +25,19 @@ def remind_sits_to_add_participants
     end
   end
 
+  # This timestamp was generated by finding the last user with request_for_details_sent_at: nil,
+  # after automation this request for details is sent automatically
+  AUTOMATION_LAUNCH_TIME = Time.zone.parse("2021-09-17 09:45:33.284979000 +0000").freeze
```

Comment:
Could this just be `3.days.ago`? In theory, anyone covered by automation would have an email sent within the last three days, right?

Reply:
Because we want to run this twice (two batches of approx 5000), we need this so that we don't send to the same set of users again the second time round.

Reply:
How come? Does this not also schedule these users for automatic chasers?

Reply:
Support want a limit on the amount of emails we send a week so they aren't overwhelmed.

Regarding the automatic chasers for these users: my understanding is that it does not, and a series of chasers are sent starting from user creation using `ParticipantDetailsReminderJob`. I can't answer "Should it?" as I wasn't involved in that work, but it seems to be by design.